lib/airtable/entity/table.rb

```diff
@@ -0,0 +1,123 @@
+module Airtable
+  module Entity
+    # Airtable Table entity
+    class Table
+      PAGE_SIZE         = 100
+      DEFAULT_DIRECTION = 'asc'.freeze
+
+      def initialize(base, name)
+        @name = name
+        @base = base
+        @args = [@base, @name]
+      end
+
+      def select(options = {})
+        params = {}
```

Comment:
Lets add support for `view` (should be a string if present) and `filterByFormula` (should be a string if present) parameters.

Reply:
Ah, yes it will be added. I want to finished with current codebase and functionality first.

Reply:
Done